ssh-setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+- name: Get IP Address of Atomic Host
+  hosts: localhost
+  become: yes
+  become_method: sudo
+
+  tasks:
+  - name: Wait for 10s to Get the IP
+    pause:
+      seconds: 10
+
+  - name: Get the IP
+    shell: virsh domifaddr atomic-node | tail -2| awk '{print $4}' | cut -d/ -f1
+    register: vm_ip
+
+  - debug: msg="{{ vm_ip.stdout }}"
+
+  - name: ssh-copy-id to the Atomic host
+    shell: ssh-copy-id -i ~/.ssh/id_rsa.pub atomic-user@$"{{ vm_ip.stdout }}"
+
+  - name: Create Inventory file
+    file:
+      path: ./inventory
+      state: touch
+
+  - name: Write Atomic host in Inventory file
```

Comment:
@gbraad I still believe we will need the inventory file. 
Because the IP of Atomic host that I'm generating from [ssh-setup.yml](https://github.com/trishnaguha/build-atomic-host/blob/master/ssh-setup.yml) comes under the scope of `localhost`. 
So that registered variable of that scope is not supposed to work if I use that variable to a new scope where `hosts: '{{ vm_ip.stdout }}'`.